src/TeamSpeak3QueryApi/Notifications/ChannelDeleted.cs

```diff
@@ -1,4 +1,4 @@
-namespace TeamSpeak3QueryApi.Net.Specialized.Notifications
+namespace TeamSpeak3QueryApi.Net.Notifications
```

Comment:
Why was this moved?

Reply:
I've removed the specialized folder cause i couldn't do anything with it

Reply:
It is a concrete implementation of the notification that the server sends and therefore belongs to the `Specialized` namespace.

Reply:
Okey sure... the notifications is part of the query and should be moved there